src/terminal/adapter/adaptDispatchGraphics.cpp

```diff
@@ -12,6 +12,19 @@
 using namespace Microsoft::Console::VirtualTerminal;
 using namespace Microsoft::Console::VirtualTerminal::DispatchTypes;
 
+// Inspired from RETURN_IF_WIN32_BOOL_FALSE
+// WIL doesn't include a RETURN_IF_FALSE, and RETURN_IF_WIN32_BOOL_FALSE
+//  will actually return the value of GLE.
+#define RETURN_IF_FALSE(b)                    \
+    do                                        \
+    {                                         \
+        BOOL __boolRet = wil::verify_bool(b); \
+        if (!__boolRet)                       \
+        {                                     \
+            return b;                         \

```

Comment:
```suggestion
            return __boolRet;                         \
```

Per what Duncan Horn said in the upstream issue.